src/components/pages/document/show/index.tsx

```diff
@@ -77,7 +83,7 @@ export default function DocumentShowComponent({ document }: DocumentShowProps) {
             commentFunc={axAddDocumentComment}
           />
         </Box>
-        <Sidebar d={document} />
+        <Sidebar d={document} speciesGroups={speciesGroups} habitatList={habitatList} />
```

Comment:
Could pls rename the "d" prop for the Sidebar component to meaningful name

Reply:
Thanks for pointing this out updated!